scripts/pull_retry.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -xuo pipefail
+#
+# Use this script to avoid image pull getting stuck once MicroShift tries to
+# do so in runtime.
+#
+
+function pull_image() {
+    local -r image=$1
+    local rc=0
+    for _ in $(seq 3); do
+        sudo crictl pull "${image}" && return
+        rc=$?
+        sleep 1
+    done
+    exit ${rc}
+}
+
+if [ $# -lt 1 ] ; then
+    echo "Usage: $(basename "$0") [images_to_download]..."
+    exit 1
+fi
+
+for img in "$@"
+do
```

Comment:
```suggestion
for img in "$@"; do
```
nit for consistency with line 11

Reply:
Done